components/scream/src/physics/share/tests/physics_saturation_unit_tests.cpp

```diff
@@ -107,7 +107,11 @@ struct UnitWrap::UnitTest<D>::TestSaturation
     //: C::macheps is machine epsilon for single or double precision as appropriate. This will be
     //multiplied by a condition # to get the actual expected numerical uncertainty.
 
-    static constexpr Scalar tol = C::macheps;
+    static constexpr Scalar tol = C::macheps
```

Comment:
This might be personal, but I think having `ifdef`'s cutting through an expression is a bit hard to parse. I think having

```
#ifndef NDEBUG
static constexpr Real tol = blah;
#else
static constexpr Real tol = blah*10000;
#endif
```
is easier to parse.